admin_app/src/app/page.tsx

```diff
@@ -109,6 +112,39 @@ export default function Home() {
     setShowEditModal(true);
   };
 
+  const onContentChange = (content_text: string) => {
```

Comment:
Inspired by your modal and wanting to make the delete and edit/add consistent, I refactored the edit/add bits. 
See what you think